src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/AbstractCloudFoundryDeployer.java

```diff
@@ -127,6 +130,21 @@ Path getApplication(AppDeploymentRequest request) {
 	}
 
 	/**
+	 * Return the Path to manifests.
+	 *
+	 */
+	Path getManifestsPath(AppDeploymentRequest request) {
+		try {
+			String manifestsPath = request.getDefinition().getProperties().get("manifests-path");
```

Comment:
I am trying to check if we can use `deployment` property option instead of this being `definition` property. Given the `CloudFoundryDeploymentProperties` are per server, I wonder if there is a way to use `per app` level property for CF deployment properties.